test/examples_tools.py

```diff
@@ -24,6 +24,16 @@ def tmp_dir(newdir):
         shutil.rmtree(newdir)
 
 
+def replace_in_file(filename, source, target):
```

Comment:
I already did a similar one in my PR, different name, won't conflict

Reply:
Ok, I'll use them then